src/objecttypes/api/schema.py

```diff
@@ -2,7 +2,34 @@
 
 from drf_yasg import openapi
 
-description = """An API to access Object types"""
+description = """An API to manage Object types.
+
+# Introduction
+
+An OBJECTTYPE represents a collection of objects of similar form and/or function.
+An OBJECTTYPE includes a definition of objects, represented as a JSON-schema, and
+metadata attributes. Metadata are stores on the high level, JSON schema is prone
+to changes and is stored in VERSIONs (see below).
```

Comment:
```suggestion
metadata attributes. Metadata is stored on the top level and the JSON schema itself is stored
in VERSIONs because the definition of an object can change over time.
```